bin/elixir

```diff
@@ -221,7 +231,7 @@ if [ -n "$RUN_ERL_PIPE" ]; then
   mkdir -p "$RUN_ERL_PIPE"
   mkdir -p "$RUN_ERL_LOG"
   ERL_EXEC="run_erl"
-  set -- "$ERTS_BIN$ERL_EXEC" -daemon "$RUN_ERL_PIPE/" "$RUN_ERL_LOG/" "$ESCAPED"
+  set -- "$ERTS_BIN$ERL_EXEC" $CERL_ARGS -daemon "$RUN_ERL_PIPE/" "$RUN_ERL_LOG/" "$ESCAPED"
```

Comment:
Not sure if this makes a difference here because we are setting ERL_EXEC="run_erl" just above.